emulators/krunkit/Portfile

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+# Upstream build source (until more official): https://github.com/slp/homebrew-krunkit/blob/main/Formula/krunkit.rb
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cargo_fetch 1.0
+
+name                krunkit
+version             0.1.4
+revision            0
+
+categories          emulators
+license             Apache-2
+platforms           darwin
+maintainers         {@tsujp wz.ht:jc+macports} openmaintainer
+
+description         Tool to launch configurable virtual machines using the libkrun platform.
+long_description    Bar for now.
+
+homepage            https://github.com/containers/krunkit
+
+# Only arm64 as libkrun only supports Hypervisor.framework on arm64.
+supported_archs     arm64
+
+checksums           ${distname}${extract.suffix} \
+                    rmd160  e7d180e5ec342f9672bfe8efa5c0e8453f4d0802 \
+                    sha256  30998931a859b4853adb1d3e66ec5072a92edb9d3e46ad69c41a647584c619a0 \
+                    size    18686
+
+depends_run         port:libkrun
+
+use_configure       no
+
+github.setup        containers ${name} ${version} v
+github.tarball_from archive
```

Comment:
these two lines should be moved to the top. It will set "version" and "name" from here so these separate lines can be removed.